extension/source/background/AggregatorController.ts

```diff
@@ -65,43 +65,31 @@ export default class AggregatorController {
         this.ethersProvider,
       );
 
-      const userAction = await this.transactionsController
-        .InternalRpc()
-        .requestTransaction(...params);
-
-      if (userAction === TransactionStatus.APPROVED) {
```

Comment:
No longer testing whether `userAction` is approved, since `requestTransaction` now throws if the user doesn't approve.